src/components/Order.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = ({orderType}) => {
+  const [orders, setOrders] = useState(0);
+  const orderOne = () => {
+    setOrders((orders) => orders + 1);
+  };
+  return (
+    <li>
```

Comment:
so because this is all wrapped in <li> you are having your ordertype/ orders and your button on the same line. Is this how you would like it to look?